lib/_http_common.js

```diff
@@ -105,7 +105,10 @@ function parserOnHeadersComplete(versionMajor, versionMinor, headers, method,
     skipBody = parser.onIncoming(parser.incoming, shouldKeepAlive);
   }
 
-  return skipBody;
+  if (typeof skipBody !== 'number')
```

Comment:
If anyone is curious - this is for compatibility with external code. (like node-spdy)